engine/src/main/java/org/terasology/rendering/world/ChunkMeshUpdateManager.java

```diff
@@ -133,6 +135,8 @@ public void shutdown() {
 
     private static class ChunkUpdateTask implements ChunkTask {
 
+        private final int numberOfMeshSegments = CoreRegistry.get(Config.class).getSystem().getVerticalChunkMeshSegments();
```

Comment:
Due to previous discussion with Immortius, I couldn't set this as a static variable, as we cannot be sure if the CoreRegistry will have the right entry by the time this class is touched.